presto-parquet/src/main/java/com/facebook/presto/parquet/reader/PageReader.java

```diff
@@ -45,13 +53,17 @@ public PageReader(CompressionCodecName codec,
             DictionaryPage compressedDictionaryPage)
             throws IOException
     {
-        this(codec, compressedPages, compressedDictionaryPage, null);
+        this(codec, compressedPages, compressedDictionaryPage, null, null, null, -1, -1);
     }
 
     public PageReader(CompressionCodecName codec,
                       LinkedList<DataPage> compressedPages,
                       DictionaryPage compressedDictionaryPage,
-                      OffsetIndex offsetIndex)
+                      OffsetIndex offsetIndex,
+                      BlockCipher.Decryptor blockDecryptor,
```

Comment:
I think it might be better to use Optional for these two

Reply:
sure